packages/input-stepper/src/LionInputStepper.js

```diff
@@ -10,14 +10,11 @@ import { IsNumber, MinNumber, MaxNumber } from '@lion/form-core';
 // @ts-expect-error false positive for incompatible static get properties. Lit-element merges super properties already for you.
 export class LionInputStepper extends LionInput {
   static get styles() {
-    return [
-      super.styles,
-      css`
-        .input-group__container > .input-group__input ::slotted(.form-control) {
-          text-align: center;
-        }
-      `,
-    ];
+    return css`
```

Comment:
Forgot why the parent does not have any styles. Everything inheriting from FormControlMixin should add `super.styles` right?

Reply:
We still don't have it. It misses styles right now. I would rather go for:

```
  static get styles() {
    return [
      // @ts-expect-error
      super.styles,
      css`
        .input-group__container > .input-group__input ::slotted(.form-control) {
          text-align: center;
        }
      `,
    ];
  }
```